docs/tutorials/mac_experimental.md

```diff
@@ -0,0 +1,99 @@
+# Using podman-machine on MacOS (x86_64 and Apple silicon)
+
+## Setup
+
+You must obtain a compressed tarball that contains the following:
+* a qcow image
+* a podman binary
+* a gvproxy binary
+
+You must also have installed brew prior to following this process.  See https://brew.sh/ for
+installation instructions.
+
+Note: If your user has admin rights, you can ignore the use of `sudo` in these instructions.
+
+
+1. Install qemu from brew to obtain the required runtime dependencies.
+
+   ```
+   brew install qemu
+   ```
+
+2. If you are running MacOS on the Intel architecture, you can skip to step 8.
+3. Uninstall the brew package
+
+   ```
+   brew uninstall qemu
+   ```
+
+4. Get upstream qemu source code.
+
+   ```
+   git clone https://github.com/qemu/qemu
```

Comment:
Coming from a newby Mac user, how do I install git, curl, or other packages that are required but are not part of the default MacOS install?

Reply:
and where's the window that I run this from?

Reply:
Installing brew will pick up the Xcode cli tools with git and friends. You'll need the Terminal.app inside /Applications/Utilities/